examples/go-client/README.md

```diff
@@ -20,17 +17,70 @@ Auth API clients must perform two-way authentication using x509 certificates:
 2. They must offer their x509 certificate, which has been previously issued
    by the auth sever.
 
-Start up a teleport auth server and then run the following commands to create an `api-user` with a signed certificate. Make sure to run this from the `go-client` directory for proper output.
+There are a few simple ways to generate the certificates needed to authenticate the client, each of which is detailed in this demo.
+
+### Authorization
+
+The Server will authorize requests for the user associated with the certificates used to authenticate the client. 
+
+Therefore, to use the API client, you need to create a user and any roles it may need for your use case. The client will act as that user and have access to everything defined by the user's role(s).
```

Comment:
```suggestion
Therefore, to use the API client, you need to create a user and any roles it may need for your use case. The client will act on behalf of that user and have access as defined by the user's role(s).
```